repo2docker/buildpacks/r.py

```diff
@@ -68,33 +64,29 @@ def r_version(self):
         version.
         """
         version_map = {
-            "3.4": "3.4",
-            "3.5": "3.5.3-1bionic",
-            "3.5.0": "3.5.0-1bionic",
-            "3.5.1": "3.5.1-2bionic",
-            "3.5.2": "3.5.2-1bionic",
-            "3.5.3": "3.5.3-1bionic",
-            "3.6": "3.6.3-1bionic",
-            "3.6.0": "3.6.0-2bionic",
-            "3.6.1": "3.6.1-3bionic",
-            "4.0": "4.0.5-1.1804.0",
-            "4.0.2": "4.0.2-1.1804.0",
-            "4.1": "4.1.2-1.1804.0",
+            "4.2": "4.2.0",
+            "4.1": "4.1.3",
+            "4.0": "4.0.5",
+            "3.6": "3.6.3",
+            "3.5": "3.5.3",
+            "3.4": "3.4.0",
+            "3.3": "3.3.3",
         }
+
         # the default if nothing is specified
-        r_version = "4.1"
+        r_version = "4.1.0"
```

Comment:
Should this be 4.1.3? The version map for 4.1 points to 4.1.3 now and previously pointed to 4.1.2. The previous version map used the latest available 4.1 but now the latest available version is 4.2, so if the policy is to use the latest, we could consider bumping it to 4.2.0 instead as well.

One change should be done here though, not using an older patch version for 4.1

Reply:
Hmmm, below there is logic related to `self._r_version` and `r_version`, I think that makes us use the version map if you set it to 4.1, so maybe stop providing the patch version here and rely on the version map?

Reply:
done!

I want to bump the default to 4.2 too, but in another PR